crates/forge/src/main.rs

```diff
@@ -148,6 +148,10 @@ struct TestArgs {
     /// Number of maximum steps during a single test. For fuzz tests this value is applied to each subtest separately.
     #[arg(long)]
     max_n_steps: Option<u32>,
+
+    /// Specify Scarb features to enable.
```

Comment:
I don't think we should refer to those as `Scarb features` should user know if those are scarb thing or cairo thing?

Reply:
Had no better idea honestly, rust doesn't have any special name for it either, just calls it "features"

Reply:
I think we should call them "features" not "scarb features"